src/no/runsafe/UserControl/database/PlayerDatabase.java

```diff
@@ -191,6 +227,7 @@ public boolean IsFirstSession(IPlayer player)
 		return GetPlayerLogout(player) == null;
 	}
 
+	private final UUID consoleUUID = UUID.fromString("00000000-0000-0000-0000-000000000000");

```

Comment:
Move this constant to another class and make it public so can use it also in PlayerUsernameLog